src/views/Projects/index.jsx

```diff
@@ -50,22 +73,32 @@ export default class Projects extends Component {
     return (
       <Fragment>
         <header className={classes.header}>
```

Comment:
Let's switch to using the AppBar component from Material UI. We can change the position prop to absolute so that it spans the full width.

```jsx
<AppBar position="absolute" className={classes.header}>
  <Typography variant="display2" align="center" color="primary">
    Codetribute
  </Typography>
  <div className={classes.flexContainer}>
    <Typography
      className={classes.highlightedText}
      variant="subheading"
      align="center">
      Find your first code contribution with Mozilla
    </Typography>
  </div>
  <SearchBox
    value={this.state.searchTerm}
    onChange={this.handleTextInputChange}
  />
</AppBar>
```